### PR TITLE
First pass at visual regression testing

### PR DESCRIPTION
**Pull request by ataylorme**

### CODE REVIEW

**Comment by ataylorme** [comment]
Multidev  created successfully! [https://pr-62-.pantheonsite.io/](https://pr-62-.pantheonsite.io/)

**Comment by ataylorme** [comment]
Multidev  created successfully! [https://pr-62-pantheon-wp-best-practices.pantheonsite.io/](https://pr-62-pantheon-wp-best-practices.pantheonsite.io/)

**Comment by ataylorme** [comment]
Visual regression test passed! Visual Regression Test Result

**Comment by ataylorme** [comment]
Visual regression test passed! Visual Regression Test Result

**Comment by ataylorme** [comment]
Visual regression test passed! Visual Regression Test Result

**Comment by ataylorme** [comment]
Visual regression test passed! Visual Regression Test Result

**Comment by ataylorme** [comment]
Visual regression test passed! [Visual report](https://circleci.com/api/v1.1/project/github/ataylorme/Advanced-WordPress-on-Pantheon/428/artifacts/0/tmp/circle-artifacts.Vk5TlXi/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test passed! [Visual report](https://circleci.com/api/v1.1/project/github/ataylorme/Advanced-WordPress-on-Pantheon/429/artifacts/0/tmp/circle-artifacts.b9CzPan/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test passed! [Visual report](https://circleci.com/api/v1.1/project/github/ataylorme/Advanced-WordPress-on-Pantheon/430/artifacts/0/tmp/circle-artifacts.0w8bOqT/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test passed! [Visual report](/tmp/circle-artifacts.NdmySmm/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test failed! [Visual report](https://circleci.com/api/v1.1/project/github/ataylorme/Advanced-WordPress-on-Pantheon/447/artifacts/0/tmp/circle-artifacts.CySCme9/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test failed! [Visual report](https://circleci.com/api/v1.1/project/github/ataylorme/Advanced-WordPress-on-Pantheon/448/artifacts/0/tmp/circle-artifacts.ozxXwkJ/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test failed! [Visual report](https://--gh.circle-artifacts.com/0/tmp/circle-artifacts.zoICjji/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/ataylorme/Advanced-WordPress-on-Pantheon/450/artifacts/0//tmp/circle-artifacts.ZGXQTp5/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/ataylorme/Advanced-WordPress-on-Pantheon/451/artifacts/0//tmp/circle-artifacts.WojQETT/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test passed! [Visual report](https://circleci.com/gh/ataylorme/Advanced-WordPress-on-Pantheon/452/artifacts/0//tmp/circle-artifacts.hudzUPp/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test passed! 
[Visual report](https://circleci.com/gh/ataylorme/Advanced-WordPress-on-Pantheon/453/artifacts/0//tmp/circle-artifacts.HTGy0j4/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/ataylorme/Advanced-WordPress-on-Pantheon/454/artifacts/0//tmp/circle-artifacts.LJlXS5D/backstop_data/html_report/index.html)

**Comment by ataylorme** [comment]
@schrapel looking at adding in visual regression results to pull requests. Would love to get your thoughts and a quick review.

**Comment by tobeycodes** [comment]
Will have a look over the weekend. Looks great though

**Comment by ataylorme** [comment]
Addresses #29

File: bin/visual-regression-test.sh [hunk 1]
```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Color Variables
+BUILD_DIR=$(pwd)
```
Comment (tobeycodes) [inline bin/visual-regression-test.sh]: An enhancement (unrelated to this PR) would be to move all this reused bash stuff into a helper file that is always sourced

File: bin/visual-regression-test.sh [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+# Variables
+BUILD_DIR=$(pwd)
+GITHUB_API_URL="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
+
+# Authenticate with Terminus
+./bin/terminus_auth.sh
+
+# Check if we are NOT on the master branch and this is a PR
+if [[ $CIRCLE_BRANCH != "master" && -n "$CI_PULL_REQUEST" ]]
```
Comment (tobeycodes) [inline bin/visual-regression-test.sh]: You could do the reverse and say 

If not {
 exit;
}

continue the rest of the file.

That means we are not indented for the rest of the file by one level

**Review by tobeycodes — APPROVED** [review]
This looks great. I really like it

**Pull request closed without merging**